shops/urls.py

```diff
@@ -1,5 +1,7 @@
 from django.urls import path
 
-urlpatterns = [
+from shops.views import CartView
 
+urlpatterns = [
+    path('/carts', CartView.as_view()),
```

Comment:
CartsView 혹은 CartListView로 class 명을 수정해주세요~

```suggestion
    path('/carts', CartsView.as_view()),
```